CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.6.0] - 2021-02-10
```

Comment:
It's better to have these notes on the Unreleased section because once we merge it, we can release it and it will automatically change this file (:

Reply:
Oh ok thanks, so like this: https://github.com/vtex/io-clients/pull/35/commits/0f708ff2299c97e0253e9bd5c3e5394d2c2131eb?